features.md

```diff
@@ -118,3 +118,76 @@ mods --delete='naturals' --delete='a2e2'
 
 Keep in mind that these operations are not reversible.
 You can repeat the delete flag to delete multiple conversations at once.
+
```

Comment:
could you remove this extra line, please?

Reply:
There's https://github.com/DavidAnson/markdownlint, and if you use VS Code you can use their [extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) and it renders a yellow wiggle underline that you can either hover over or list in the "problems" view, or cycle through problems with F8, like this:
![image](https://github.com/user-attachments/assets/9c92cec1-d522-42c7-bd9d-4751ecbd56a9)